packaging/templates/puppynet/sentry/validator/bor/config.toml

```diff
@@ -25,7 +25,7 @@ syncmode = "full"
     # debug = true
 
 [p2p]
-    maxpeers = 1
+    maxpeers = 20
```

Comment:
here max peer will be 20 only.. can it have more peers than this.. like our own 9 validator.. 10 sentry if we extend 3 and seeds ?

Reply:
This is for validator, 20 is anyway too much for val